dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/graph/AbstractSuccessorParser.java

```diff
@@ -15,19 +15,23 @@
  * limitations under the License.
  */
 
-package org.apache.dolphinscheduler.server.master.event;
+package org.apache.dolphinscheduler.server.master.engine.graph;
 
-/**
- * This exception represent the exception can not recover, this happens when the event is broken.
- * And when we get this exception, we will drop the event.
- */
-public class StateEventHandleError extends Exception {
+import org.apache.dolphinscheduler.dao.entity.TaskInstance;
+
+public abstract class AbstractSuccessorParser implements ISuccessorParser {
```

Comment:
Can you add some comments for the concept of `Successor Parser`?

Reply:
I removed this class, this is replaced by 
```java
/**
 * Used to adjust the flow of tasks.
 * <p> Only {@link ConditionLogicTask} and {@link SwitchLogicTask} need to adjust the flow of tasks.
 */
@Slf4j
@Component
public class SuccessorFlowAdjuster {

}
```

which is used to control the flow of workflow and adjust the post tasks which should be trigger or skip.